privacyidea/lib/task/simplestats.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+#
+#  2018-08-06 Paul Lettich <paul.lettich@netknights.it>
+#
+#
+# This code is free software; you can redistribute it and/or
+# modify it under the terms of the GNU AFFERO GENERAL PUBLIC LICENSE
+# License as published by the Free Software Foundation; either
+# version 3 of the License, or any later version.
+#
+# This code is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU AFFERO GENERAL PUBLIC LICENSE for more details.
+#
+# You should have received a copy of the GNU Affero General Public
+# License along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#
+import logging
+
+from privacyidea.lib.tokenclass import TOKENKIND
+from privacyidea.lib.token import get_all_token_users, get_tokens
+from privacyidea.lib.monitoringstats import write_stats
+from privacyidea.lib.subscriptions import get_users_with_active_tokens
+from privacyidea.lib.task.base import BaseTask
+# from privacyidea.lib.user import get_user_list
+from privacyidea.lib import _
+
+__doc__ = """This is a statistics task which collects simple statistics from the database."""
```

Comment:
Add info: If you want to use more possible keys simply add them to the options method and add a corresponding property.